projects/plugins/jetpack/extensions/shared/memberships.js

```diff
@@ -1,5 +1,9 @@
 /* global tb_show, tb_remove */
 
+import { getJetpackData } from '@automattic/jetpack-shared-extension-utils';
+import { useEntityProp } from '@wordpress/core-data';
+import { __ } from '@wordpress/i18n';
```

Comment:
I noticed a [warning](https://github.com/Automattic/jetpack/actions/runs/5742658272/job/15565234375?pr=32201#step:9:5138) that I'm not sure was there before:

```
[plugins/jetpack 3:12.045] [build-production-extensions] Block view script dependency check detected issues!

[plugins/jetpack 3:12.045] [build-production-extensions] In projects/plugins/jetpack/_inc/blocks/subscriptions/view.asset.php
[plugins/jetpack 3:12.045] [build-production-extensions] Dependency `lodash` should not be used by the subscriptions block's view.js.
[plugins/jetpack 3:12.045] [build-production-extensions] Dependency `wp-compose` should not be used by the subscriptions block's view.js.
[plugins/jetpack 3:12.045] [build-production-extensions] Dependency `wp-core-data` should not be used by the subscriptions block's view.js.
[plugins/jetpack 3:12.045] [build-production-extensions] Dependency `wp-element` should not be used by the subscriptions block's view.js.
[plugins/jetpack 3:12.045] [build-production-extensions] Dependency `wp-plugins` should not be used by the subscriptions block's view.js.

[plugins/jetpack 3:12.045] [build-production-extensions] Dependencies allowed may be adjusted by editing the arrays at the top of projects/plugins/jetpack/tools/check-block-assets.php.
```

Reply:
That would definitely be a blocker, we do not want to increase the size of the frontend's bundle.

Reply:
I would've expected build process to get rid of thoes for the `view` since any of the imported functions aren't actually used there, but it depends how the webpack is configured and if "tree shaking" applies.